Formatter+SFLoggingKit.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Formatter+SFLoggingKit.swift
+//  SFLoggingKit
+//
+//  Created by Aleksandar Sergeev Petrov on 23.11.21.
+//
+
+import Foundation
+
+/// Default log timestamp formatter sugar extension
+public extension Formatter {
+
+    static var logTimestampFormatter: DateFormatter = {
+        var formatter = DateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd HH:mm:ss.SSS"
```

Comment:
[optional]

Do you think it is a good idea to pass the format as an external argument so it can be configured on project level?